buildkite/bazelci.py

```diff
@@ -1117,7 +1117,7 @@ def execute_commands(
                     bazel_version,
                     bazel_binary,
                     platform,
-                    build_flags,
+                    build_flags + ["--embed_label=%s" % get_release_name_from_branch_name()] if save_but else [],
```

Comment:
There is a bug since this line is equivalent to

(build_flags + ["--embed_label=%s" % get_release_name_from_branch_name()]) if save_but else []

(note the parentheses)., i.e. build_flags won't be passed if save_but is False.

Reply:
Yes, I figured out this 😂 It is fixed at HEAD now. Thanks!

Reply:
Ah, I didn't check the other commits. I just remembered that we ran into the exact same problem some time ago